modules/developing/pages/clients/flink.adoc

```diff
@@ -0,0 +1,476 @@
+= Apache Flink
+
+Apache Flink is an open source stream processing framework with powerful stream- and batch-processing capabilities.
+{product_name} offers a fully managed Apache Flink service enabling you to build and run streaming applications without having to manage the underlying infrastructure.
+
+== Connect to Flink
+
+The {product_name} Flink endpoint is currently available at `https://pulsar-aws-useast2.api.dev.streaming.datastax.com/flink/api/`.
+To connect, you'll need both an AstraCS token and a Pulsar token.
+
+For more on generating the Astra and Pulsar JWT tokens, see xref:getting-started:index.adoc[].
+For more on the differences between the Astra and Pulsar tokens, see xref:operations:onboarding-faq.adoc#secure-sign-on-roles-and-permissions[].
+For more on setting up your Pulsar binaries, see xref:configure-pulsar-env.adoc[].
```

Comment:
"...setting up your Pulsar binaries".  I felt a little bit confused in the beginning about this statement (since AS is a managed service). Then I realized that this is for client setup. Maybe it is better to clarify a bit that this is for the purpose of running Pulsar CLI tools on the client side.